dev_guide/templates.adoc

```diff
@@ -97,15 +150,15 @@ by piping both commands:
 $ oc process -f <filename.json> | oc create -f -
 ----
 
-You can override any link:../dev_guide/templates.html#parameters[parameters]
-defined in the JSON file by adding the `-v` option followed by a
+You can override any link:../dev_guide/templates.html#parameters[parameter]
+values defined in the JSON file by adding the `-v` option followed by a
```

Comment:
parameter's value defined in the JSON/YAML file, by adding...

or without JSON/YAML would be even better